types/sharedb/index.d.ts

```diff
@@ -2,6 +2,7 @@
 // Project: https://github.com/share/sharedb
 // Definitions by: Steve Oney <https://github.com/soney>
 //                 Eric Hwang <https://github.com/ericyhwang>
+//                 Eve Shum <https://github.com/eveshum>
```

Comment:
By putting your name and username here, the bot will start tagging you on all PRs for the sharedb types.

Do you feel comfortable enough with the sharedb type definitions to review and approve PRs on them?

Reply:
omg im going to remove myself...